### PR TITLE
Securing sql server access with aad-pod-binding

### PR DESCRIPTION
**Pull request by mahluk**
Accessing sql server without credentials in code level by using Azure Managed Identities binded in pod level.

### CODE REVIEW

**Review by savas-guven — APPROVED** [review]

**Comment by yufufi** [comment]
Test

File: _posts/2019-12-19-aad-pod-binding-sql-server-conn.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
+---
+layout: post
+title: Securing Access to SQL Server with Managed Identities and aad-pod-binding
+author: haluk.aktas@deepnetwork.com
```
Comment (yufufi) [inline _posts/2019-12-19-aad-pod-binding-sql-server-conn.md:4]: Please use author defined in `_data`

File: _posts/2019-12-19-aad-pod-binding-sql-server-conn.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+
+Create User Assigned Managed Identity and note the `clientId` from output for later use:
+
+```
+az identity create -g myResourceGroup -n myidentity -o json
+```
+
+Install created identity to our cluster by deploying the following:
+
+```
```
Comment (yufufi) [inline _posts/2019-12-19-aad-pod-binding-sql-server-conn.md:100]: Add `yaml` for syntax highlighting.

image

image

**Review by yufufi — COMMENTED** [review]
Pleas send a new PR as this is already merged.

**Pull request merged**